docs/src/extend/custom-formatters.md

```diff
@@ -29,37 +35,17 @@ module.exports = async function(results) {
 };
 ```
 
-To run ESLint with this formatter, you can use the `-f` (or `--format`) command line flag:
+To run ESLint with this formatter, you can use the [`-f` (or `--format`)](../use/command-line-interface#f---format) command line flag:
```

Comment:
```suggestion
To run ESLint with this formatter, you can use the [`-f` (or `--format`)](../use/command-line-interface#-f---format) command line flag:
```

Reply:
thx. interestingly the version i did was what the VSCode markdown autocomplete does, and works w/in the files. but upon checking your suggestion, you're right.